crates/api/src/local_user/ban_person.rs

```diff
@@ -68,10 +64,10 @@ impl Perform for BanPerson {
       expires,
     };
 
-    ModBan::create(context.pool(), &form).await?;
+    ModBan::create(&mut conn, &form).await?;
 
     let person_id = data.person_id;
-    let person_view = PersonView::read(context.pool(), person_id).await?;
+    let person_view = PersonView::read(&mut conn, person_id).await?;
```

Comment:
The code is not very nice with `&mut` everywhere. Maybe you could add a method `context.conn()` which can replace `context.pool()` directly?

Reply:
`context.pool()` is now replaced with `context.conn().await?`.

The functions now take `impl DbConn`, which makes it possible to use `&mut *conn` instead. This would be needed for transactions.